.cbmc-batch/source/openssl/ec_override.c

```diff
@@ -0,0 +1,246 @@
+/*
```

Comment:
Duplicate code. Skipping in favor of https://github.com/aws/aws-encryption-sdk-c/pull/392/.